From 83619333fc5c7913c07b665bbe4d19fe4ff9f820 Mon Sep 17 00:00:00 2001
From: Mrinmoy Ghosh <mrghosh@cisco.com>
Date: Tue, 19 Aug 2025 23:18:29 +0000
Subject: [PATCH] doc: zebra: Doc update for zebra option kernel-mac-ext-learn

Document that --kernel-mac-ext-learn signals zebra to operate in MAC
external-learn mode without duplicating the broader option documentation
introduced by the preceding production patch.

Signed-off-by: Mrinmoy Ghosh <mrinmoy_g@hotmail.com>
Signed-off-by: Mrinmoy Ghosh <mrghosh@cisco.com>
Signed-off-by: Tamer Ahmed <tamerahmed@microsoft.com>
(cherry picked from commit 12d3632f8677004ea7f14cece117cb7ed8478a91)
---
 doc/user/zebra.rst | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/doc/user/zebra.rst b/doc/user/zebra.rst
index 81fbbc9682..cc9f9e1452 100644
--- a/doc/user/zebra.rst
+++ b/doc/user/zebra.rst
@@ -126,6 +126,8 @@ Besides the common invocation options (:ref:`common-invocation-options`), the
 
    **Global option** that enables hardware-based MAC learning and aging mode.
 
+   This option signals to zebra that it is operating in MAC external-learn mode.
+
    This is a system-wide zebra option (not EVPN-MH specific) designed for
    platforms with hardware ASIC capabilities that perform MAC learning and
    aging in hardware rather than kernel software.
-- 
2.34.1

